bili.config.ts

```diff
@@ -6,12 +6,17 @@ const config: Config = {
       tsconfigOverride: {
         include: ['src', 'ethers']
       }
-    }
+    },
   },
-
+  extendConfig: (config) => ({
```

Comment:
@dwojno Why the need to add `extendConfig` here with the empty `externals` array? Was this needed in order to be able to use `bundleNodeModules`?

Reply:
Because the default externals are the package.json dependencies and you can only bundle node_modules that is not a dependency, so I need to clear out all externals, to bundle `nats.ws` package